src/Microsoft.OpenApi.OData.Reader/Generator/OpenApiEdmTypeSchemaGenerator.cs

```diff
@@ -147,6 +147,8 @@ public static OpenApiSchema CreateSchema(this ODataContext context, IEdmPrimitiv
                 OneOf = null,
                 AnyOf = null
             };
+
+            string format;
```

Comment:
format is never being assigned?

Reply:
They are being used within the use case blocks: https://github.com/microsoft/OpenAPI.NET.OData/blob/5e64822e4d90dda71e8d86571e3cb7af3cb99e5b/src/Microsoft.OpenApi.OData.Reader/Generator/OpenApiEdmTypeSchemaGenerator.cs#L172